<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="6" failures="0" disabled="0" errors="0" time="0" timestamp="2021-12-31T12:11:48.944" name="AllTests">
  <testsuite name="boost_exception_test_suite" tests="6" failures="0" disabled="0" skipped="0" errors="0" time="0" timestamp="2021-12-31T12:11:48.944">
    <testcase name="test_catch_go_boost_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
    <testcase name="test_catch_go_boost_exception_as_boost_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
    <testcase name="test_catch_derived_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
    <testcase name="test_catch_derived_exception_as_go_boost_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
    <testcase name="test_catch_derived_exception_as_boost_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
    <testcase name="test_catch_other_exception" status="run" result="completed" time="0" timestamp="2021-12-31T12:11:48.944" classname="boost_exception_test_suite" />
  </testsuite>
</testsuites>
